lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -478,185 +490,227 @@ class PullbackInfo {
     }
   }
 
-  /// Creates an enum declaration with the given VJP generic signature, whose
+  /// Creates an enum declaration with the given VJP/JVP generic signature, whose
   /// cases represent the predecessors of the given original block.
   EnumDecl *
-  createBasicBlockPredecessorEnum(SILBasicBlock *originalBB,
-                                  SILAutoDiffIndices indices,
-                                  CanGenericSignature vjpGenericSig) {
+  createBasicBlockEnum(SILBasicBlock *originalBB,
```

Comment:
This term is inconsistent with earlier suggestions. Please rename to `createBranchingTraceDecl`.